src/CrowdedMod/Patches/CreateGameOptionsPatches.cs

```diff
@@ -155,4 +155,37 @@ public static bool Prefix()
             return false;
         }
     }
-}
\ No newline at end of file
+
+    [HarmonyPatch(typeof(CreateOptionsPicker), nameof(CreateOptionsPicker.SetImpostorButtons))]
+    public static class CreateOptionsPicker_SetImpostorButtons
+    {
+        public static bool Prefix(CreateOptionsPicker __instance, int numImpostors)
+        {
+            IGameOptions targetOptions = __instance.GetTargetOptions();
+            targetOptions.SetInt(Int32OptionNames.NumImpostors, numImpostors);
+            __instance.SetTargetOptions(targetOptions);
+            __instance.UpdateImpostorsButtons(numImpostors);
+
+            return false;
+        }
+    }
+
+    [HarmonyPatch(typeof(CreateOptionsPicker), nameof(CreateOptionsPicker.SetMaxPlayersButtons))]
+    public static class CreateOptionsPicker_SetMaxPlayersButtons
+    {
+        public static bool Prefix(CreateOptionsPicker __instance, int maxPlayers)
+        {
+            if (DestroyableSingleton<FindAGameManager>.InstanceExists)
+            {
+                return true;
+            }
+
+            IGameOptions targetOptions = __instance.GetTargetOptions();
+            targetOptions.SetInt(Int32OptionNames.MaxPlayers, maxPlayers);
+            __instance.SetTargetOptions(targetOptions);
+            __instance.UpdateMaxPlayersButtons(targetOptions);
+
+            return false;
+        }
+    }
```

Comment:
instead of reimplementing entire methods to bypass OOB, you should've overwritten the array itself to have more elements. Basically just port [this](https://github.com/CrowdedMods/CrowdedMod/blob/fefec2a57c8dec11be771fb08a8b752fd9193aed/src/CrowdedMod/CrowdedModPlugin.cs#L26-L27) to v8. Overwriting methods is not acceptable (unless there is no other way) because it leads to mod conflicts and CrowdedMod aims to be compatible with ***everything***

Reply:
> instead of reimplementing entire methods to bypass OOB, you should've overwritten the array itself to have more elements. Basically just port [this](https://github.com/CrowdedMods/CrowdedMod/blob/fefec2a57c8dec11be771fb08a8b752fd9193aed/src/CrowdedMod/CrowdedModPlugin.cs#L26-L27) to v8. Overwriting methods is not acceptable (unless there is no other way) because it leads to mod conflicts and CrowdedMod aims to be compatible with _**everything**_

Based on the older pr, seems like he did this to improve compatibility with other mods like tohe

Reply:
well the part for createoptionspicker has no reason to exist

Reply:
I agree. Seems like u only need to patch max players and max impostors

Reply:
They add buttons and register new max imp max players in this part

Reply:
Removed these patches